docs/Makefile

```diff
@@ -223,3 +224,7 @@ dummy:
 	$(SPHINXBUILD) -b dummy $(ALLSPHINXOPTS) $(BUILDDIR)/dummy
 	@echo
 	@echo "Build finished. Dummy builder generates no files."
+
+.PHONY: serve
+serve:
+	sphinx-autobuild -b html $(ALLSPINXOPTS) --watch ../i3ipc . $(BUILDDIR)/html
```

Comment:
Should `sphinx-autobuild` be added as an optional dev/docs dependency? Otherwise, I'd be happy to remove it.